app/assets/v2/js/pages/bounty_details.js

```diff
@@ -398,6 +406,13 @@ window.addEventListener('load', function() {
 
 
 $(document).ready(function(){
+
+    $("body").delegate('a[href="/express_interest"]', 'click', function(e){
+        alert("Please leave a comment on the pull request to express interest in performing the work. \n\nRedirecting you now..")
```

Comment:
Should this tell the user to comment on the issue versus pull request?

Reply:
either or, i've been using interchangably.  shuld i not do that anymore?

Reply:
Whatever you want to do xD - I was only saying that because it'd probably take them to an issue versus a PR for a new bounty request.

Reply:
i'm agnostic here